webdriver/tests/bidi/browsing_context/capture_screenshot/capture_screenshot.py

```diff
@@ -35,3 +37,33 @@ async def test_capture(bidi_session, top_context, inline, compare_png_bidi):
 
     comparison = await compare_png_bidi(new_data, data)
     assert comparison.equal()
+
+
+@pytest.mark.parametrize("delta_width", [-10, +20])
```

Comment:
Actually would you mind to move this test to it's own file eg `viewport.py` in this folder? That way I could also already prepare easily a metadata file for us.

Reply:
Added